tests/AutoSuggestBox.js

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corp.  All Rights Reserved. Licensed under the MIT License. See License.txt in the project root for license information.
+
+describe("AutoSuggestBox control directive tests", function () {
+    var scope,
+        compile;
+
+    beforeEach(angular.mock.module("winjs"));
+    beforeEach(angular.mock.inject(function ($rootScope, $compile) {
+        scope = $rootScope.$new();
+        compile = $compile;
+    }));
+
+    function initControl(markup) {
+        var element = angular.element(markup)[0];
+        document.body.appendChild(element);
+        var compiledControl = compile(element)(scope)[0];
+        scope.$digest();
+        return compiledControl;
+    }
+
+    it("should initialize a simple autosuggestbox", function () {
+        var compiledControl = initControl("<win-auto-suggest-box></win-auto-suggest-box>");
+        
+        expect(compiledControl.winControl).toBeDefined();
+        expect(compiledControl.winControl instanceof WinJS.UI.SearchBox);
```

Comment:
Instead of checking if this is an instance of WinJS.UI.SearchBox, this should be expect(compiledControl.winControl instanceof WinJS.UI.AutoSuggestBox);